app/pages/reservation/reservation-information/ReservationInformation.js

```diff
@@ -87,13 +87,13 @@ class ReservationInformation extends Component {
         'billingEmailAddress',
       ];
 
+      // ensure fields are always shown if payable
       if (isPayableAmount) {
         formFields = [...formFields, ...paymentFields];
```

Comment:
Shouldn't this be outside the `if` statement, as we want to display the fields regardless of the price?

Reply:
If `billingFirstName` etc are in the required list, they will be in these fields already. 

This is the case where payment is required AND the billing fields are not present in the metadata, so we have the fields needed to process payment.

Reply:
Ah, right I think get it now. So if the fields are in the metadata, they are included in `formFields` by default. So this adds them if the fields are not in the metadata but it's a payable amount.

Reply:
Or well, they'll be in `requiredFields` if they're in the metadata. Still, all good then.

Reply:
Yes. I'm not sure which edge cases that might happen, as they will probably set the correct metadata set for all billing fields, so this might be revisited later.

Reply:
Yep